src/Resources/config/command.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
+        <service id="Sonata\NewsBundle\Command\SwaggerDocblockConvertCommand" class="Sonata\NewsBundle\Command\SwaggerDocblockConvertCommand" public="true">
```

Comment:
IMO, this command doesn't belong to this package.

Reply:
I use it to convert API. My bad.